torch/onnx/experimental/fonnx/v1/exporter.py

```diff
@@ -0,0 +1,41 @@
+import io
+
+import torch
+
+from typing import Union
+
+
+def dynamo_export(
+    torch_model: torch.nn.Module,
```

Comment:
Just like having a custom op as a `Callable`, I feel that having a model as `Callable` would broaden the use access so much that it would certainly create more export bugs. 

Although dynamo allows exporting functions and models, I believe we should be more restrictive, at least for now, and just go with model. When the FX acquisition APIs are more stable and with higher coverage, we could consider expanding ONNX support too.

what do you think?

Reply:
sg